webpack.config.js

```diff
@@ -2,7 +2,7 @@ const NodemonPlugin = require('nodemon-webpack-plugin');
 const nodeExternals = require('webpack-node-externals');
 
 const baseConfig = {
-  mode: 'development',
+  mode: 'production',
```

Comment:
Should this be set to `process.env.NODE_ENV`?

Reply:
yes @scarney81 - good catch